docs/help/account/user-account-settings.md

```diff
@@ -0,0 +1,28 @@
+# User account settings
+
+Accessible from the Dashboard, the Account Settings page allows you to quickly modify certain elements related to your account.
+
+From this page, you can modify:
+
+- your first and last name
+- your password
+- your username
+
+## Modify Username
+
+Your username serves also as your slug, which is part of the URL of your documentation and hubs except if you use custom domains.
```

Comment:
```suggestion
Your username serves also as your Slug, which is part of the URL of your documentation and hubs except if you use custom domains.
```
Slug is a proper name and take a capital letter.

Reply:
I disagree, we don't need an upcase here. An example here, on GitHub documentation: https://docs.github.com/en/enterprise-cloud@latest/graphql/reference/queries#enterprise

Reply:
👍